OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -169,8 +177,24 @@ protected override void Created(Actor self)
 			base.Created(self);
 		}
 
+		void UpdateCondition(Actor self)
+		{
+			if (conditionManager == null)
```

Comment:
`if (string.IsNullOrEmpty(Info.ReloadingCondition) || conditionManager == null)`!!!

Right now we crash when anything except the V2 fires its weapon.